lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -48,7 +48,7 @@
       - When passing a hashed password it must be generated with the format
         C('str["md5"] + md5[ password + username ]'), resulting in a total of
         35 characters. An easy way to do this is C(echo "md5$(echo -n
-        'verysecretpasswordJOE' | md5sum)").
+        'verysecretpasswordJOE' | md5sum | awk 'print $1')").
```

Comment:
I believe the `awk` command needs `{}`: `awk '{print $1}'`

Reply:
You are right, my bad.